pypit/arsort.py

```diff
@@ -978,7 +985,7 @@ def write_setup(setup_dict, use_json=False):
     # Write
     setup_file, nexist = get_setup_file()
     if nexist == 1:
-        msgs.error("Cannot over-write existing setup file.  If you wish to, remove and rerun")
```

Comment:
Would it be worthwhile checking if "output overwrite" is set to True before overwriting?